specification/security/resource-manager/Microsoft.Security/preview/2019-01-01-preview/assessments.json

```diff
@@ -261,13 +261,29 @@
         },
         "links": {
           "$ref": "#/definitions/AssessmentLinks"
+        },
+        "metadata": {
+          "$ref": "./assessmentMetadata.json#/definitions/SecurityAssessmentMetadataProperties"
+        },
+        "partnersData": {
+          "$ref": "#/definitions/SecurityAssessmentPartnerData"
         }
       },
       "required": [
         "resourceDetails",
         "status"
       ]
     },
+    "SecurityAssessmentPartnerData": {
+      "type": "object",
+      "description": "Data regarding 3rd party partner integration",
+      "properties": {
+        "secret": {
+          "type": "string",
+          "description": "secret to authenticate the partner"
```

Comment:
annotate with "x-ms-secret" : "true"

Reply:
Ignore if you get error 'invalid schema' .. I thought x-ms-secret support is already added.. 

---
In reply to: [367704883](https://github.com/Azure/azure-rest-api-specs/pull/8143#discussion_r367704883) [](ancestors = 367704883)

Reply:
I also cannot find a doc on that (to understand what it means exactly), can you please share?

Reply:
added the property but I still cannot find any documentation and not sure what will this do to our clients